### ISSUE TITLE
com.fasterxml.jackson.databind.exc.InvalidFormatException

### ISSUE THREAD

**Issue by ChrisMnemo** [binance-exchange/binance-java-api#286]
**Android Studio / Intellij Android: API from mavenCentral:** com.github.binance-exchange:binance-java-api:master-SNAPSHOT

When I try to get the candlestick data via:
`List<Candlestick> candlesticks = client.getCandlestickBars("NEOETH", CandlestickInterval.WEEKLY);`

I get the following error:
```
FATAL EXCEPTION: DefaultDispatcher-worker-3
Process: com.example.and_candletest, PID: 12058
com.binance.api.client.exception.BinanceApiException: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Long` from String "0.09700000": not a valid Long value at [Source: okhttp3.ResponseBody$BomAwareReader); line: 1, column: 17] (through reference chain: java.util.ArrayList[0]->com.binance.api.client.domain.market.Candlestick["closeTime"])
```
This error only occurs when building for Android. The method works when building for Windows. Tried with Java and Kotlin. All the other methods work.

**Comment by GregSino** [comment]
Same error but also for windows for:

`List<Order> allOrders = client.getAllOrders(new AllOrdersRequest("BTCUSDT").limit(10));`

after changing:

`public static final long DEFAULT_RECEIVING_WINDOW = 6_000_000L;`
to:
`public static final long DEFAULT_RECEIVING_WINDOW = 5_000L;`

in BinanceApiConstant, due to the error  with the limit of the recvWindow:
`recvWindow cannot exceed 60000` 

Absolutely don't how we can resolve this problem.

**Comment by ChrisMnemo** [comment]
Yes, sounds like the same problem.

**Comment by ChrisMnemo** [comment]
**Android Studio:** API Local
I imported the API into Android Studio for further testing.
I used @JsonIgnore to deactivate the 3 functions that use parameters of type Long in Candelstick.java:
```
@JsonIgnore
  public void setOpenTime(Long openTime) {
    this.openTime = openTime;
  }
@JsonIgnore
  public void setCloseTime(Long closeTime) {
    this.closeTime = closeTime;
  }
  @JsonIgnore
  public void setNumberOfTrades(Long numberOfTrades) {
    this.numberOfTrades = numberOfTrades;
  }
```
**After these steps I am able to retrive the candlestick data in mobile phone**. The problem is that the order in the array for the values gets mixed up: 
openTime = null (@JsonIgnore), 
open = low, 
high = open, 
low = high, 
close(String) = openTime(Long), 
etc.) 
I guess this is also the reason for the format Exception for jackson databind. After removing all the functions that feed long parameter, only functions that hand over strings as parameter are left and so no Exception occurs. 
What could be the reason that this happens when building for Android, but not for Windows?
